src/Run/Setup.cs

```diff
@@ -410,20 +410,16 @@ private string GetTool(Command commandToExecute, string os, string configPath, L
         private string GetProject(Command commandToExecute, List<string> parametersSelectedByUser)
         {
             string project = string.Empty;
-            bool moreThanOneProject = false;
-            foreach (string param in parametersSelectedByUser)
+            
+            if(parametersSelectedByUser != null)
             {
-                if (commandToExecute.Alias[param].Settings.TryGetValue("Project", out project))
+                if (parametersSelectedByUser.Count(p => commandToExecute.Alias[p].Settings.TryGetValue("Project", out project)) > 1)
```

Comment:
Use .Any instead of .Count as that will stop after the first one as opposed to actually counting all of them.